src/components/AvatarCropModal/AvatarCropModal.js

```diff
@@ -0,0 +1,258 @@
+import PropTypes from 'prop-types';
+import React, {memo, useCallback, useEffect} from 'react';
+import {Image, Pressable, View} from 'react-native';
+import {GestureHandlerRootView} from 'react-native-gesture-handler';
+import {
+    interpolate,
+    useAnimatedGestureHandler,
+    useAnimatedStyle,
+    useSharedValue,
+} from 'react-native-reanimated';
+import {SafeAreaProvider} from 'react-native-safe-area-context';
+import CONST from '../../CONST';
+import compose from '../../libs/compose';
+import colors from '../../styles/colors';
+import styles from '../../styles/styles';
+import variables from '../../styles/variables';
+import Button from '../Button';
+import HeaderWithCloseButton from '../HeaderWithCloseButton';
+import Icon from '../Icon';
+import * as Expensicons from '../Icon/Expensicons';
+import Modal from '../Modal';
+import Text from '../Text';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../withWindowDimensions';
+import ImageCropView from './components/ImageCropView';
+import Slider from './components/Slider';
+import imageManipulator from './libs/imageManipulator';
+
+const propTypes = {
+    /** Link to image for cropping   */
+    imageUri: PropTypes.string,
+
+    /** Callback to be called when user closes the modal */
+    onClose: PropTypes.func,
+
+    /** Callback to be called when user crops image */
+    onCrop: PropTypes.func,
```

Comment:
I think this should be named as onSave or onSubmit. If I am not wrong, this callback will be used when user submits the image.

Reply:
Renamed to onSave